File: 0.0.0.0_8080/#0.0.0.0/csp/hello.scala
// ============================================================================
// C servlet sample for the G-WAN Web Application Server (http://trustleap.ch/)
// ----------------------------------------------------------------------------
// hello.scala: G-WAN now talks Scala
// ============================================================================
import api.Gwan

object hello
{
  def jmain(env: Long, args: Array[String]) : Int = 
  {
     Gwan.xbufCat(Gwan.getReply(env), "Hello World");
     return 200;
  }
}
// ============================================================================
// End of Source Code
// ============================================================================

File: 0.0.0.0_8080/#0.0.0.0/csp/hello.c
// ============================================================================
// C servlet sample for the G-WAN Web Application Server (http://trustleap.ch/)
// ----------------------------------------------------------------------------
// hello.c: just used with ab (ApacheBench) to benchmark a minimalist servlet
// ============================================================================
#include "gwan.h" // G-WAN exported functions

int main(int argc, char *argv[])
{
   static char msg[] = "Hello, ANSI C!";
   xbuf_ncat(get_reply(argv), msg, sizeof(msg) - 1);
   
   return 200; // return an HTTP code (200:'OK')
}
// ============================================================================
// End of Source Code
// ============================================================================

